dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -200,7 +203,14 @@ public static Tool convertEntryToTool(Entry container, DockstoreWebserviceConfig
             // ensure that descriptor is non-null before adding to list
             if (!toolVersion.getDescriptorType().isEmpty()) {
                 // do some clean-up
-                toolVersion.setMetaVersion(String.valueOf(version.getLastModified() != null ? version.getLastModified() : new Date(0)));
+                if(isDockstoreTool) {
+                    io.dockstore.webservice.core.Tag castedTag = (io.dockstore.webservice.core.Tag)version;
+                    toolVersion.setMetaVersion(String.valueOf(castedTag.getLastBuild() != null ? castedTag.getLastBuild() : new Date(0)));
```

Comment:
Hmm, this will set date to 1970 if lastBuild is not set. Is that what you want?

Reply:
The logic was written like this before, although I don't really know the reasoning.